napari_cellseg3d/code_plugins/plugin_model_inference.py

```diff
@@ -281,6 +281,7 @@ def _set_tooltips(self):
         )
 
         thresh_desc = (
+            "NOT RECOMMENDED ON FIRST RUN - check results without first!\n"
```

Comment:
Will this be shown every time or only on the first run?

Reply:
It's shown when you hover over the threshold option in the inference UI; perhaps I can just check if it has been run once already ?